jolie/src/main/java/jolie/CommandLineParser.java

```diff
@@ -559,6 +568,11 @@ public CommandLineParser( String[] args, ClassLoader parentClassLoader, Argument
 				i++;
 				charset = argsList.get( i );
 				optionsList.add( argsList.get( i ) );
+			} else if( "--serviceIdentifier".equals( argsList.get( i ) ) ) {
```

Comment:
you probably meant --serviceId ? (The help string above says so.)

Reply:
serviceIdentifier